contributed_definitions/nyaml/NXem_eels.yaml

```diff
@@ -6,24 +6,27 @@ symbols:
     Number of electron energy loss bins.
 type: group
 NXem_eels(NXem_method):
-  # NXem_method can offers to keep a SPECTRUM_SET
   # NXem_method also has an NXprocess which in this base class can be
   # specialized to include EELS-specific post-processing
-  SPECTRUM_SET(NXspectrum_set):
+  (NXspectrum_set):
     doc: |
       NXspectrum_set_em specialized for EELS.
-    stack(NXdata):
-      # \@signal: data_counts
-      # \@axes: [axis_y, axis_x, axis_energy_loss]
-      # \@energy_loss_indices: 2
-      # \@axis_x_indices: 1
-      # \@axis_y_indices: 0
+    # \@signal:, \@axes: \@energy_loss_indices \@axis_x_indices:
+    stack_oned(NXdata):
+      axis_energy_loss(NX_NUMBER):
+        unit: NX_ENERGY
+        dim: (n_energy_loss,)
+        \@long_name(NX_CHAR):
+          doc: |
+            Energy loss.
+    stack_oned(NXdata):
       axis_energy_loss(NX_NUMBER):
         unit: NX_ENERGY
         dim: (n_energy_loss,)
         \@long_name(NX_CHAR):
           doc: |
             Energy loss.
```

Comment:
You have the `stack_oned` twice now, that can't be right.

Reply:
That was a mistake, has been resolved